### ISSUE TITLE
Versions app loads JS file that does not exist

### ISSUE THREAD

**Issue by nickvergessen** [owncloud/core#18867]
```json
{"reqId":"9S3IEx34LPA+6bWAyLBy","remoteAddr":"","app":"files_versions","message":"Could not find resource file \"\/apps\/files_versions\/js\/versions.js\"","level":3,"time":"2015-09-07T08:10:01+00:00"}
```

https://github.com/owncloud/core/blob/310d79728447ecf69f18d0b61a527397bd961888/apps/files_versions/appinfo/app.php#L22
```php
OCP\Util::addscript('files_versions', 'versions');
```

But the directory does not know that file anymore:
https://github.com/owncloud/core/tree/310d79728447ecf69f18d0b61a527397bd961888/apps/files_versions/js

I guess it was removed when moving to the rightside bar by @PVince81

**Comment by MorrisJobke** [comment]
> I guess it was removed when moving to the rightside bar by @PVince81

https://github.com/owncloud/core/pull/18748/files#diff-e82735adf3e23a1648cd449aa65725b7 Yep. I will create a PR

**Comment by MorrisJobke** [comment]
Fix is in #18868
